Host/Program.cs

```diff
@@ -21,6 +22,15 @@
     .AddEntityFrameworkStores<ClinicContext>();
 
 
+//Start ConfigureAutoMapper
+var config = new AutoMapper.MapperConfiguration(cfg =>
+{
+    cfg.AddProfile(new AutoMapperProfile());
```

Comment:
also just suggestion, we can pass reference to WebApi assembly where our profiles are stored,
it will search Profiles in this assembly
`cfg.AddMaps(typeof(WebApi.AutoMapper.UserProfile));`